tensorflow/python/distribute/distribute_lib.py

```diff
@@ -633,19 +638,26 @@ class InputOptions(
       strategy.experimental_distribute_dataset(
           dataset,
           tf.distribute.InputOptions(
-              experimental_prefetch_to_device=False)))
+              experimental_prefetch_to_device=False,
+              replication_mode=InputReplicationMode.PER_WORKER)))
```

Comment:
Can we call this `experimental_replication_mode` since it is a new option?

Reply:
changed.